data/themes/default/style.css

```diff
@@ -114,9 +113,15 @@ PianoRoll {
 	qproperty-gridColor: rgb( 128, 128, 128 );
 	qproperty-noteModeColor: rgb( 255, 255, 255 );
 	qproperty-noteColor: rgb( 119, 199, 216 );
-	qproperty-barColor: #4afd85;
 	qproperty-noteBorderRadiusX: 5;
 	qproperty-noteBorderRadiusY: 2;
+	qproperty-selectedNoteColor: rgb( 0, 64, 192 );
+	qproperty-barColor: #4afd85;
```

Comment:
> I don't know if its worth adding [python] a dependency just for CSS.

Well, it's pre-installed on Ubuntu as well as OS X and we've added obscure build dependencies for less (#1835, da65f62). :smiling_imp: